src/run/runner/wall_time/perf.rs

```diff
@@ -0,0 +1,63 @@
+use crate::run::runner::helpers::setup::run_with_sudo;
+use codspeed::fifo::FifoIpc;
```

Comment:
Have we given thought abouth whether the `FifoIPC` belong to `codpseed_rust` or to the codspeed runner ?

My initial impression is that it should be the runner that is the source of truth about what's hapenning in terms of FIFO, but this question may have an obvious answer once we tackle other languages than rust :thinking: